src/Elastic.Apm/Logging/ConsoleLogger.cs

```diff
@@ -1,13 +1,61 @@
-﻿using System;
+using System;
+using System.IO;
 
 namespace Elastic.Apm.Logging
 {
-	internal class ConsoleLogger : AbstractLogger
+	internal class ConsoleLogger : IApmLogger
 	{
-		private ConsoleLogger(LogLevel level) : base(level) { }
+		private readonly TextWriter _errorOut;
+		private readonly LogLevel _level;
+		private readonly TextWriter _standardOut;
 
-		internal static ConsoleLogger Instance { get; } = new ConsoleLogger(LogLevelDefault);
+		public ConsoleLogger(LogLevel level, TextWriter standardOut = null, TextWriter errorOut = null)
+		{
+			_level = level;
+			_standardOut = standardOut ?? Console.Out;
+			_errorOut = standardOut ?? Console.Error;
+		}
 
-		protected override void PrintLogLine(string logLine) => Console.WriteLine(logLine);
+		protected internal static LogLevel DefaultLogLevel { get; } = LogLevel.Error;
+		public static ConsoleLogger Instance { get; } = new ConsoleLogger(DefaultLogLevel);
+
+		public void Log<TState>(LogLevel level, TState state, Exception e, Func<TState, Exception, string> formatter)
+		{
+			//TODO: Discuss UtcNow?
```

Comment:
I'd prefer UtcNow... with logs from multiple processes on multiple machines in the cloud not using utc would make people's life harder.

Reply:
Yes, I will take a look at it today and update.

Reply:
Agreed, `UtcNow` all the way.